website/docs/guides/1.15.0-upgrade-guide.html.markdown

```diff
@@ -0,0 +1,52 @@
+---
+layout: "mongodbatlas"
+page_title: "MongoDB Atlas Provider 1.15.0: Upgrade and Information Guide"
+sidebar_current: "docs-mongodbatlas-guides-1150-upgrade-guide"
+description: |-
+MongoDB Atlas Provider 1.15.0: Upgrade and Information Guide
+---
+
+# MongoDB Atlas Provider 1.15.0: Upgrade and Information Guide
+
+The Terraform MongoDB Atlas Provider version 1.15.0 has a number of new and exciting features.
+
+**New Resources, Data Sources, and Features:**
+
+
+**Breaking Changes:**
+
+Format of `okta_idp_id` that contains the IdP Id in `federated_settings_identity_provider` resource now accepts a different format to align with Atlas API. Both the current and new IdP Id format are accepted in terraform-provider 1.15.0 version.
+
+***WARNING:*** Old IdP Id format will no longer be accepted starting 1.16.0 and onwards. We recommend to update to the new format as soon as possible. A warning will appear if old Id is still being used. Follow the guide below to start using the new Id format.
+
+The old format was a 20-hex characters long string.
+
+The new format is a 24-hex characters long string.
+
+Update guide: 
+
+1. Get new 24-hex characters string ID value
+	- Option 1: Use Atlas API v20231001, both of those endpoints include the new `id` in the response:
```

Comment:
what do you mean with `both of those endpoints`?

Reply:
maybe `both of the following endpoints`?

Reply:
You are right. Fixed